sonaranalyzer-dotnet/its/expected/akka.net/Akka.TestKit-{0D3CBAD0-BBDB-43E5-AFC4-ED1D3ECDC224}-S4136.json

```diff
@@ -0,0 +1,238 @@
+"location":  [
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\EventFilter\IEventFilterApplier.cs",
+"region":  {
+"startLine":  44,
+"startColumn":  14,
+"endLine":  44,
+"endColumn":  20
+}
+},
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\EventFilter\IEventFilterApplier.cs",
+"region":  {
+"startLine":  93,
+"startColumn":  11,
+"endLine":  93,
+"endColumn":  17
+}
+}
+]
+},
+{
+"id":  "S4136",
+"message":  "All 'ExpectOne' signatures should be adjacent",
+"location":  [
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\EventFilter\Internal\EventFilterApplier.cs",
+"region":  {
+"startLine":  32,
+"startColumn":  21,
+"endLine":  32,
+"endColumn":  30
+}
+},
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\EventFilter\Internal\EventFilterApplier.cs",
+"region":  {
+"startLine":  58,
+"startColumn":  18,
+"endLine":  58,
+"endColumn":  27
+}
+}
+]
+},
+{
+"id":  "S4136",
+"message":  "All 'Expect' signatures should be adjacent",
+"location":  [
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\EventFilter\Internal\EventFilterApplier.cs",
+"region":  {
+"startLine":  42,
+"startColumn":  21,
+"endLine":  42,
+"endColumn":  27
+}
+},
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\EventFilter\Internal\EventFilterApplier.cs",
+"region":  {
+"startLine":  68,
+"startColumn":  18,
+"endLine":  68,
+"endColumn":  24
+}
+}
+]
+},
+{
+"id":  "S4136",
+"message":  "All 'CompareTo' signatures should be adjacent",
+"location":  [
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\TestActorRefBase.cs",
+"region":  {
+"startLine":  143,
+"startColumn":  20,
+"endLine":  143,
+"endColumn":  29
+}
+},
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\TestActorRefBase.cs",
+"region":  {
+"startLine":  183,
+"startColumn":  36,
+"endLine":  183,
+"endColumn":  45
+}
+}
+]
+},
+{
+"id":  "S4136",
+"message":  "All 'Equals' signatures should be adjacent",
+"location":  [
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\TestActorRefBase.cs",
+"region":  {
+"startLine":  133,
+"startColumn":  30,
+"endLine":  133,
+"endColumn":  36
+}
+},
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\TestActorRefBase.cs",
+"region":  {
+"startLine":  148,
+"startColumn":  21,
+"endLine":  148,
+"endColumn":  27
+}
+},
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\TestActorRefBase.cs",
+"region":  {
+"startLine":  188,
+"startColumn":  36,
+"endLine":  188,
+"endColumn":  42
+}
+}
+]
+},
+{
+"id":  "S4136",
+"message":  "All 'Tell' signatures should be adjacent",
+"location":  [
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\TestActorRefBase.cs",
+"region":  {
+"startLine":  71,
+"startColumn":  21,
+"endLine":  71,
+"endColumn":  25
+}
+},
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\TestActorRefBase.cs",
+"region":  {
+"startLine":  94,
+"startColumn":  21,
+"endLine":  94,
+"endColumn":  25
+}
+},
+{
+"uri":  "sources\akka.net\src\core\Akka.TestKit\TestActorRefBase.cs",
+"region":  {
+"startLine":  195,
```

Comment:
`void ICanTell.Tell(object message, IActorRef sender)` - should we treat [explicit interface](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation) implementations the same?